STANDARD.md

```diff
@@ -233,13 +233,13 @@ Each file's *root* group (path `/`) must further define the attributes:
         - `filename_%T.h5` (without file system directories)
       - for `groupBased`: (fixed value)
         - `/data/%T/` (must be equal to and encoded in the `basePath`)
-      - for `stepBased`: (fixed value)
+      - for `variableBased`: (fixed value)
         - data-format internal convention
         - *slowest varying index* of data
 
-### `stepBased` Encoding of Iterations
+### `variableBased` Encoding of Iterations
 
-In order to correlate openPMD iterations with an index of data-format internal updates/steps or an index in the slowest varying dimension of an array, the *root* group (path `/`) must contain an additional variable once `stepBased` is chosen for `iterationEncoding`:
+In order to correlate openPMD iterations with an index of data-format internal updates/steps or an index in the slowest varying dimension of an array, the iteration base path (default: path `/default`) must contain an additional variable once `variableBased` is chosen for `iterationEncoding`:
```

Comment:
It's `/default`? That is a but curious...

Reply:
Ah, I confused `default` with `data`. At least got the first letter right there..